fahrschein-http-jdk11/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'fahrschein.java-conventions'
+    id 'fahrschein.maven-publishing-conventions'
+}
+
+dependencies {
+    api project(':fahrschein-http-api')
+    testImplementation(testFixtures(project(':fahrschein-http-test-support')))
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_11
+java.targetCompatibility = JavaVersion.VERSION_11
+
+publishing.publications.maven.pom.description = 'Fahrschein HTTP Client using JDK11\'s HttpClient'
```

Comment:
Wouldn't it in this case be better to use the `"` quotations to get rid of quoting the `'`?

Reply:
👍